pom.xml

```diff
@@ -46,6 +49,12 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>org.roaringbitmap</groupId>
+            <artifactId>RoaringBitmap</artifactId>
+            <version>0.6.66</version>
```

Comment:
latest version is `0.8.11`. Any reason you picked this version?

Reply:
Newer versions are incompatible with the Java version of the REST framework (Jersey) that Bitbucket supports. It's a strange case of version compatibility, but anything past 0.7 crashes with an obscure exception. 🤔